content/articles/getting-started-with-neural-network-in-pytorch/index.md

```diff
@@ -0,0 +1,287 @@
+Neural networks are an algorithmic set of rules for discovering patterns in data. Using a neural network model will help us understand how our brains process information in the same way. The networks are constructed from discrete pieces that resemble neurons and are often called "neurons" or "units." There are a certain number of weighted inputs for each unit. 
```

Comment:
When you talk of the same, way what way ; mentioned before;  are you referring to?